### PR TITLE
Quiz

### PR DESCRIPTION
**Pull request by jkelly8823**

### CODE REVIEW

File: app/views/quizzes/edit.html.erb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<h1>Editing quiz</h1>
```
Comment (kunalvudathu) [inline app/views/quizzes/edit.html.erb]: we don't need this right? if so, can you remove this file?

File: app/controllers/quizzes_controller.rb [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+    disp_quiz()
+  end
+
+  # GET /quizzes/new
+  def new
+    @quiz = Quiz.new
+  end
+
+  # GET /quizzes/1/edit
+  def edit
```
Comment (kunalvudathu) [inline app/controllers/quizzes_controller.rb]: remove this call, update rts.txt

File: app/controllers/quizzes_controller.rb [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+      else
+        format.html { render :new, status: :unprocessable_entity }
+        format.json { render json: @quiz.errors, status: :unprocessable_entity }
+      end
+    end
+  end
+
+  # PATCH/PUT /quizzes/1 or /quizzes/1.json
+  def update
+    respond_to do |format|
```
Comment (kunalvudathu) [inline app/controllers/quizzes_controller.rb]: no need for update

File: app/controllers/quizzes_controller.rb [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+      else
+        format.html { render :edit, status: :unprocessable_entity }
+        format.json { render json: @quiz.errors, status: :unprocessable_entity }
+      end
+    end
+  end
+
+  # DELETE /quizzes/1 or /quizzes/1.json
+  def destroy
+    @quiz.destroy
```
Comment (kunalvudathu) [inline app/controllers/quizzes_controller.rb]: do we need to destroy? no right?

Comment (jkelly8823) [reply app/controllers/quizzes_controller.rb]: Correct, I'll remove that

File: app/controllers/quizzes_controller.rb (hunk 3 above)
Comment (jkelly8823) [reply app/controllers/quizzes_controller.rb]: Will remove

File: app/controllers/quizzes_controller.rb (hunk 2 above)
Comment (jkelly8823) [reply app/controllers/quizzes_controller.rb]: Will remove

File: app/views/quizzes/edit.html.erb (hunk 1 above)
Comment (jkelly8823) [reply app/views/quizzes/edit.html.erb]: Will remove

File: app/controllers/quizzes_controller.rb [hunk 5]
```diff
@@ -0,0 +1,111 @@
+class QuizzesController < ApplicationController
+  before_action :authenticate_user!
+  before_action :set_quiz, only: %i[ show edit update destroy ]
```
Comment (kunalvudathu) [inline app/controllers/quizzes_controller.rb]: update this as well

**Review by kunalvudathu — APPROVED** [review]

**Pull request merged**